metadata-ingestion/src/datahub/ingestion/source/looker/looker_source.py

```diff
@@ -1104,6 +1122,80 @@ def get_workunit_processors(self) -> List[Optional[MetadataWorkUnitProcessor]]:
             ).workunit_processor,
         ]
 
+    def emit_independent_looks_mcp(
+        self, dashboard_element: LookerDashboardElement
+    ) -> Iterable[MetadataWorkUnit]:
+        yield from [
```

Comment:
use the `auto_workunit` helper here and elsewhere in the code to convert MCEs/MCPs to MetadataWorkUnits

```suggestion
        yield from auto_workunit(self._make_chart_metadata_events(
                dashboard_element=dashboard_element, dashboard=None
            ))
```

Reply:
done